package org.mixit.cesar.security.service.exception;


/**
 * Exception launched when user isn't found
 */
public class UserNotFoundException extends RuntimeException{
}
